proposals/utils/pdf_diff_sections.py

```diff
@@ -554,14 +556,22 @@ def get_row_fields(self):
         return rows
 
 
+
+
 class AttachmentSection(BaseSection):
```

Comment:
I think creating a subclass here is a good way of doing things. You want subclasses to be stable at the interface, which is the case here.

This isn't as clean as it could be because of the `Row` class. `BaseSection` is nice to subclass because it has a generic interface. But `Row` is kind of annoying because it's purpose-built for Django models, without a generic interface, which makes you work around it.

So in the future my advice would be to keep the generic interface in mind from the start. So you would have a `BaseRow` where you define how other classes should talk to it, and then a `DjangoModelRow` that implements that interface. Even if you only ever use django model rows, your classes still become easier to talk to.